Resources/doc/cache-resolver/aws_s3.rst

```diff
@@ -263,7 +263,6 @@ for the list of available options.
 Note, that the following options are configured automatically and will be
 ignored, even if you configure it via ObjectOptions:
 
-* ``ACL``
```

Comment:
you could add a note to the end of this chapter to reference the example you provide in the events documentation. users might not look at events when they try to figure out how to do signed urls with s3